app/db.js

```diff
@@ -16,6 +16,13 @@ const client = knex({
   },
 })
 
+async function addRecipe(recipe) {
```

Comment:
why async?

Reply:
I think it's the better of two evils in this context.